src/app/screens/connectors/ConnectGaloy/index.tsx

```diff
@@ -52,163 +52,17 @@ export default function ConnectGaloy(props: Props) {
-            errMessage ? `: ${errMessage}` : ""
-          }`;
-          toast.error(alertMsg);
-        }
-      } else {
-        setSmsCodeRequested(data.userRequestAuthCode.success);
-      }
-    } catch (e: unknown) {
-      console.error(e);
-      if (e instanceof Error) {
-        toast.error(`${t("galoy.errors.failed_to_request_sms")}: ${e.message}`);
-      }
-    } finally {
-      setLoading(false);
-    }
-  }
-
-  async function requestAuthToken(event: React.FormEvent<HTMLFormElement>) {
-    event.preventDefault();
-    setLoading(true);
-    const authQuery = {
-      query: `
-        mutation userLogin($input: UserLoginInput!) {
-          userLogin(input: $input) {
-            errors {
-                message
-            }
-            authToken
-          }
-        }
-      `,
-      variables: {
-        input: {
-          phone: phoneNumber,
-          code: smsCode,
-        },
-      },
-    };
-    const meQuery = {
-      query: `
-          query getinfo {
-            me {
-              defaultAccount {
-                defaultWalletId
-              }
-            }
-          }
-        `,
-    };
-    try {
-      const { data: authData } = await axios.post(url, authQuery, {
-        headers: defaultHeaders,
-        adapter: fetchAdapter,
-      });
-      if (authData.error || authData.errors) {
-        const error = authData.error || authData.errors;
-        const errMessage = error?.errors?.[0]?.message || error?.[0]?.message;
-        const errorMsg = `${t("galoy.errors.failed_to_login_sms")}${
-          errMessage ? `: ${errMessage}` : ""
-        }`;
-        throw new Error(errorMsg);
-      }
-      if (authData.data.userLogin.errors.length > 0) {
-        throw new Error(authData.data.userLogin.errors[0].message);
-      }
-      const authToken = authData.data.userLogin.authToken as string;
-
-      const { data: meData } = await axios.post(url, meQuery, {
-        headers: {
-          ...defaultHeaders,
-          Authorization: `Bearer ${authToken}`,
-        },
-        adapter: fetchAdapter,
-      });
-      if (meData.error || meData.errors) {
-        const error = meData.error || meData.errors;
-        console.error(error);
-        const errMessage = error?.errors?.[0]?.message || error?.[0]?.message;
-        const alertMsg = `${t("galoy.errors.setup_failed")}${
-          errMessage ? `: ${errMessage}` : ""
-        }`;
-        toast.error(alertMsg);
-      } else {
-        const walletId = meData.data.me.defaultAccount.defaultWalletId;
-        saveAccount({ authToken, walletId });
-      }
-    } catch (e: unknown) {
-      console.error(e);
-      if (e instanceof Error) {
-        toast.error(`${t("galoy.errors.setup_failed")}: ${e.message}`);
-      }
-    } finally {
-      setLoading(false);
+  async function loginWithJwt(event: React.FormEvent<HTMLFormElement>) {
+    if (!jwt) {
+      return null;
     }
```

Comment:
There is an error thrown if jwt is falsy below. (see line 80) This should trigger the validation message for the user. 

```suggestion
```